packages/amplication-client/src/VersionControl/PendingChange.tsx

```diff
@@ -80,6 +80,13 @@ export const changeOriginMap = {
         type: "Service Settings",
         pluralTypeName: "Service Settings",
       },
+      [models.EnumBlockType.Package]: {
+        relativeUrl: `settings/packages`,
+        icon: "settings",
```

Comment:
```suggestion
        icon: "package",
```

Reply:
@mulygottlieb  we need a new icon for this

Reply:
this icon already exists, the client already uses this in some places (it's the three stars)